PrismaDB-QueryAST/PrismaDB-QueryAST.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>0.2.8.0</Version>
+    <Version>0.1.0.0</Version>
```

Comment:
```suggestion
    <Version>1.0.0.0</Version>
```

Reply:
This looks more generic, just to keep it neat

Reply:
This is actually functional as well, for local testing it will supercede nuget packages

Reply:
Ok, other than that, looks good